test/utils/utils.test.ts

```diff
@@ -19,10 +20,43 @@ describe('earnDataParser', () => {
         const actualResult = earnDataParser(inputData)
         assert.typeOf(actualResult, expectedType)
         // Assert
-        assert.deepEqual(actualResult.rawData, expectedOutput.rawData)
         assert.deepEqual(actualResult.sortedData, expectedOutput.sortedData)
         assert.deepEqual(actualResult, expectedOutput)
       }
     })
   }
 })
+
+describe('normalizeDate', () => {
+  for (const testCase of fixture.normalizeDate) {
+    it(testCase.testDescription, () => {
+      // Arrange
+      const inputData = testCase.inputData
+      const expectedType = testCase.outputType
+      const expectedOutput = testCase.expectedOutput
```

Comment:
This could be destructured in one line:
`const { inputData, expectedType, expectedOutput } = testCase`